### ISSUE TITLE
Cannot initialize multiple interpreters

### ISSUE THREAD

**Issue by Wetbikeboy2500** [hetu-script/hetu-script#20]
The following code throws an error when calling init() on hetu2. hetu and hetu2 should be independent of each other.

```
var hetu = HTAstInterpreter();
await hetu.init(externalFunctions: {
  'hello': (List<dynamic> positionalArgs, Map<String, dynamic> namedArgs) => {'greeting': 'hello'},
});
await hetu.eval(r'''
    external fun hello
    fun main {
      var dartValue = hello()
      print('dart value:', dartValue)
      dartValue['foo'] = 'bar'
      return dartValue
    }''');

hetu.invoke('main');

var hetu2 = HTAstInterpreter();
await hetu2.init(externalFunctions: { //Error on this line
  'hello': (List<dynamic> positionalArgs, Map<String, dynamic> namedArgs) => {'greeting': 'hello'},
});
await hetu2.eval(r'''
    external fun hello
    fun main {
      var dartValue = hello()
      print('dart value:', dartValue)
      dartValue['foo'] = 'bar'
      return dartValue
    }''');

hetu2.invoke('main');
```

**Comment by Wetbikeboy2500** [comment]
https://github.com/hetu-script/hetu-script/blob/60704e6eb737c570b62c8ca6cc52788ec8733ddd/lib/src/ast/ast_parser.dart#L20

Seems like it comes from this being static. When I removed the static, the error went away. I'm not sure of the impact this has from being non-static.

**Comment by rockingdice** [comment]
Thank you for your information. The case of multiple interpreters is not fully tested. @hythl0day is currently working on vm implementation, so we don't know if it still exists with the vm version.

**Comment by hythl0day** [comment]
Solved in 0.1.0
